nservicebus/messaging/databus.md

```diff
@@ -68,6 +68,21 @@ snippet:MessageWithLargePayloadUsingConvention
 
 NServiceBus DataBus implementations currently behave differently with regard to cleanup of physical attachments used to transfer data properties. `FileShareDataBus` **does not** remove physical attachments once the message is gone. `AzureDataBus` **does** remove Azure storage blobs used for physical attachments once the message is gone.
 
+### Custom cleanup strategy
+
+Theoretically NServiceBus cannot tell when a file should be deleted. Perhaps the message is (functionally) deferred for the file to be processed later, or it is passed along to another handler. When the file is removed upon committing the transaction, the deferred or passed on message cannot process the file anymore. Removing the file depends on your functional needs.
```

Comment:
Remove your

Theoretically is redundant 

I think listing the scenarios in point for would be better